sote/game/scenes/game.lua

```diff
@@ -812,16 +814,278 @@ function gam.draw()
 		end
 		gam.planet_shader:send('tile_neighbor_realm', gam.tile_neighbor_realm_texture)
 	end
-
 	--- example of usage of recalculate_smooth_data_map
-	--[[
-	if gam.planet_shader:hasUniform("tile_sea") then
-		if gam.DATA_TEXTURES_CACHE["sea"] == nil then
-			gam.recalculate_smooth_data_map(sea, "sea")
+
+	---commenting
+	---@param a Tile
+	---@param b Tile
+	local function sample_face(a, b)
+		local _, _, f = tile.index_to_coords(a.tile_id)
+		return f / 6
+	end
+
+	-- if gam.planet_shader:hasUniform("face_id") then
+	-- 	if gam.DATA_TEXTURES_CACHE["face_id"] == nil then
+	-- 		gam.recalculate_smooth_data_map(sample_face, "face_id", nil, 0, 0)
+	-- 	end
+	-- 	gam.planet_shader:send("face_id", gam.DATA_TEXTURES_CACHE["face_id"])
+	-- end
```

Comment:
Delete these lines or if we want to keep them, provide a comment for the function and wrap them in a multiline lua comment for easier enabling/disabling:
```lua
--[[
code
--]]
```